webui/dev/index.tsx

```diff
@@ -37,20 +19,19 @@ const extensionURL = (extension: Extension) => encodeURIComponent(
 const pageSettings: PageSettings = {
     pageTitle: 'Open VSX Registry',
     listHeaderTitle: 'Extensions for VS Code Compatible Editors',
-    logoURL: '/openvsx-registry.svg',
     logoAlt: 'Open VSX Registry',
+    logoURL: '/openvsx-registry.svg',
```

Comment:
You could solve the logo color problem by declaring a property `toolbarContent: React.ReactNode` instead of `logoURL`, `logoAlt` and `toolbarText`. Then you can move `OpenVSXLogo` to the `dev` folder and use it here.

Reply:
Instead of having `ReactNode` in the interface for `pageSettings` I have set the type to `any` (which is not good) because I have a function reference that renders some JSX not an actual React Node the reason why I'm doing this is I want to pass the `darkMode` prop into the logo which is not accessible in `webui/dev/index.tsx` where the `pageSettings` are defined.
In the toolbar component Now I have this:

![image](https://user-images.githubusercontent.com/46004116/79975568-9e1e7f80-84b4-11ea-8da0-1ca3f1a71d14.png)

I don't know what can be the appropriate type for `pageSettings.toolbarContent`? 

I tried:

```ts
toolbarContent?: (props: { className: string, darkMode: boolean }) => JSX.Element
```  

because I thought it's a function with some props that renders some JSX but it does not work.

Reply:
This works:

```
toolbarContent?: React.FunctionComponent<{ className: string, darkMode: boolean }>;
```

and then use it like

```
{
    ToolbarContent ? <ToolbarContent
        darkMode={props.darkMode}
        className={props.classes.toolbarLogo}
    /> : ''
}
```